### PR TITLE
feat: generate random notifications in local development

### PR DESCRIPTION
**Pull request by drodil**
## Hey, I just made a Pull Request!

Use HTTP router to generate random notifications in local development to more easily find issues with UI and UX.

#### :heavy_check_mark: Checklist

- [x] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: plugins/notifications-common/api-report.md [hunk 1]
```diff
@@ -39,11 +39,19 @@ export type NotificationReadSignal = {
...
+// @public (undocumented)
+export const NotificationSeverities: readonly [
+  'critical',
+  'high',
+  'normal',
+  'low',
+];
+
 // @public
 export const notificationSeverities: NotificationSeverity[];
```
Comment (Rugvip) [inline plugins/notifications-common/api-report.md:51]: It already exists right here :p

**Review by Rugvip — CHANGES_REQUESTED** [review]
:+1: , just the severities bit

File: plugins/notifications-common/api-report.md (hunk 1 above)
Comment (drodil) [reply plugins/notifications-common/api-report.md:51]: Af fuck :D

**Comment by mareklibra** [comment]
I would rather go with a predictable environment with hard-coded initial data. In general, there might be a benefit of such randomization in "unplanned testing" but I do not see that reasonable here. On the contrary, a predictable environment can simplify developer's workflows.

Just sharing my opinion. I am ok with merging this anyway as long as others agree.

**Comment by drodil** [comment]
The problem with initial data is that the notifications should be created on the fly to see the actual functionality instead of having them pre-populated in the database. Having just all notifications with the title `Test notification` is not a very easy way to develop this feature in a local environment as you only get one type of notification. Generally, it's good to test with some pre-populated data but in this case, I think also random data is more than welcome to see all kinds of notifications during development.

**Comment by Rugvip** [comment]
Perhaps it makes sense to move the intervall to instead be a button in the sidebar? Med a bit more wiring in the setup with a slim http API, but hopefully not too much. Thinking you could have a few different ones at that point to either send from a fixed set or random

**Comment by drodil** [comment]
Yeah, that could be doable with small changes. I'll look at this at some point in near future!

**Review by Rugvip — APPROVED** [review]
:shipit:

**Pull request merged**